action.yml

```diff
@@ -59,13 +59,32 @@ runs:
       uses: ublue-os/remove-unwanted-software@v6
       if: ${{ inputs.maximize_build_space == 'true' }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      with:
+        install: true
+        cache-binary: ${{ inputs.use_cache }}
+
+    - name: Install BlueBuild
```

Comment:
the version of podman github runners use is older. We should try to prefer docker when working with the built in github runners

Reply:
Using the install script wouldn't be too bad either. This way the CLI repo would manage the way it should be installed.

```bash
bash <(curl -s https://raw.githubusercontent.com/blue-build/cli/main/install.sh)
```

Reply:
Looking at this again, the curl wouldn't work for the unstable cli option since I don't set it to download main in the install script ever. The docker commands should work though since we're directly copying the binary from the installer image which should match up with the actual version of the image you want.